src/apprunner/wizards/apprunnerCreateServiceWizard.ts

```diff
@@ -39,7 +39,7 @@ const validateName = (name: string) => {
         )
     }
 
-    let matches = name.match(badNameRegExp)
+    let matches: string[] | undefined = name.match(badNameRegExp) ?? undefined
```

Comment:
seems like a type interface regression?

Reply:
Previously Typescript was allowing `string[]` to be assigned in-place of `RegExpMatchArray` which is incorrect because `RegExpMatchArray` is typed as `[string, string?, string?, ...]`. The first element is always present.